ddtrace/pin.py

```diff
@@ -23,10 +23,13 @@
 
 
 class Pin(object):
-    """Pin (a.k.a Patch INfo) is a small class which is used to
-    set tracing metadata on a particular traced connection.
-    This is useful if you wanted to, say, trace two different
-    database clusters.
+    """
+    Pin (a.k.a Patch INfo) is a small class which is used to control tracing of
+    execution contexts for a particular object instance of an instrumented class
+    or for a loaded module that is instrumented.
```

Comment:
That's a mouthful!

Reply:
"control tracing of execution contexts" makes it sounds like Pins are part of context management/propagation. All Pins do for this though is enable/disable tracing on certain code paths, could we just say that specifically?